examples/index.rst

```diff
@@ -0,0 +1,19 @@
+🏃‍♀️ Runhouse Examples 🏠
+=======================
+
+Table of Contents
+-----------------
+.. toctree::
+   :maxdepth: 1
+   :caption: Use Cases
+
+   inference
+   training
+   distributed
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Examples
+
+   Llama2 on AWS EC2 <https://github.com/run-house/runhouse/tree/main/examples/llama2>
```

Comment:
@rohinb2 for the readme generation, I think it actually makes more sense to just include these as a link to the GH rather than rendering the readme directly on the webpage. the readme is only useful alongside the code files.


renders something like this in the sidebar
<img width="262" alt="Screenshot 2024-02-28 at 12 28 45 AM" src="https://github.com/run-house/runhouse/assets/16568633/d19c0dbc-1f4e-4968-94e9-932f1fbe3764">